metaquot/ppxlib_metaquot.ml

```diff
@@ -125,12 +128,18 @@ module Patt = Make (struct
 
   let annotate p core_type = ppat_constraint ~loc:core_type.ptyp_loc p core_type
 
-  let cast ext =
+  let cast ext : Ppxlib.pattern =
```

Comment:
This type annotation is not needed anymore!

```suggestion
  let cast ext =
```

Reply:
Thank you.